homeassistant/components/gdacs/strings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "config": {
+    "title": "Global Disaster Alert and Coordination System (GDACS)",
```

Comment:
We should add `already_configured` item under `abort`.

Reply:
Added, and tested.